### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

**Comment by fzaninotto** [comment]
I'm -1 about changing the version in peerDependencies. We don't need it, and it can be considered as a breaking change. We need to check the package is forward compatible, not break its backward compatibility.

File: packages/react-admin/package.json [hunk 1]
```diff
@@ -34,22 +34,22 @@
         "react-dom": "^16.9.0"
     },
     "dependencies": {
-        "@material-ui/core": "^4.3.3",
-        "@material-ui/icons": "^4.2.1",
-        "@material-ui/styles": "^4.3.3",
+        "@material-ui/core": "^4.10.0",
```
Comment (fzaninotto) [inline packages/react-admin/package.json:37]: the dependencies in react-admin should not change either.

File: yarn.lock [hunk 2]
```diff
@@ -873,13 +873,20 @@
...
 
-"@babel/runtime@7.8.4", "@babel/runtime@^7.0.0", "@babel/runtime@^7.1.2", "@babel/runtime@^7.3.1", "@babel/runtime@^7.3.4", "@babel/runtime@^7.4.0", "@babel/runtime@^7.4.4", "@babel/runtime@^7.4.5", "@babel/runtime@^7.5.5", "@babel/runtime@^7.6.2", "@babel/runtime@^7.6.3", "@babel/runtime@^7.7.2", "@babel/runtime@^7.7.4", "@babel/runtime@^7.8.3":
+"@babel/runtime@7.8.4", "@babel/runtime@^7.0.0", "@babel/runtime@^7.1.2", "@babel/runtime@^7.3.1", "@babel/runtime@^7.3.4", "@babel/runtime@^7.4.0", "@babel/runtime@^7.4.4", "@babel/runtime@^7.4.5", "@babel/runtime@^7.5.5", "@babel/runtime@^7.6.2", "@babel/runtime@^7.6.3", "@babel/runtime@^7.7.2", "@babel/runtime@^7.7.4":
   version "7.8.4"
   resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.8.4.tgz#d79f5a2040f7caa24d53e563aad49cbc05581308"
   integrity sha512-neAp3zt80trRVBI1x0azq6c57aNBqYZH8KhMm3TaB7wEI5Q4A2SHfBHE8w9gOhI/lrqxtEbXZgQIrHP+wvSGwQ==
   dependencies:
     regenerator-runtime "^0.13.2"
 
+"@babel/runtime@^7.10.0":
```
Comment (fzaninotto) [inline yarn.lock:883]: This one is troublesome. Can you check that we don't double the runtime in production bundles?

Comment (djhi) [reply yarn.lock:883]: After building the demo, I can confirm the runtime isn't bundled twice

Comment (djhi) [reply yarn.lock:883]: In fact, the final bundle is even smaller (-2.45 KB) :smile:

**Pull request merged**